e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -149,26 +159,32 @@ describe('Upload Files', () => {
     });
 
     it('MM-T341 Download link on preview - Image file (non SVG)', () => {
-        ['small-image.png', 'image-file.jpg', 'image-file.bmp', 'image-file.gif', 'image-file.tiff'].forEach((image) => {
-            cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(image);
-            cy.get('.post-image').should('be.visible');
-            cy.postMessage(MESSAGES.SMALL);
-            cy.uiWaitUntilMessagePostedIncludes(MESSAGES.SMALL);
-            cy.getLastPostId().then((postId) => {
-                cy.get(`#post_${postId}`).within(() => {
+        ['bmp-image-file.bmp', 'png-image-file.png', 'jpg-image-file.jpg', 'gif-image-file.gif', 'tiff-image-file.tif'].
```

Comment:
This particular tests bought in its own set of images, Images as per the docs (@saturninoabril google drive) were provided in this PR, i replaced those in this test and removed now redundant images from fixtures